src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Sql/Sprocs/GetExtendedQueryTagErrorsV36.sql

```diff
@@ -0,0 +1,57 @@
+/***************************************************************************************/
+-- STORED PROCEDURE
+--     GetExtendedQueryTagErrorsV36
+--
+-- FIRST SCHEMA VERSION
+--     36
+--
+-- DESCRIPTION
+--     Gets the extended query tag errors by tag path.
+--
+-- PARAMETERS
+--     @tagPath
+--         * The TagPath for the extended query tag for which we retrieve error(s).
+--     @limit
+--         * The maximum number of results to retrieve.
+--     @offset
+--         * The offset from which to retrieve paginated results.
+--
+-- RETURN VALUE
+--     The tag error fields and the corresponding instance UIDs.
+/***************************************************************************************/
+CREATE OR ALTER PROCEDURE dbo.GetExtendedQueryTagErrorsV36
```

Comment:
For the extended query tag sprocs (and in the generated workitem/qido queries), I've standardized the APIs to use `BIGINT` instead of `INT`:
<img width="1814" alt="image" src="https://github.com/microsoft/dicom-server/assets/2955179/964f598c-6f72-4c25-a3c3-b5eb713b717f">